WasteWar/Assets/Scripts/Camera Controls/Move.cs

```diff
@@ -0,0 +1,40 @@
+﻿using UnityEngine;
+
+public class Move : MonoBehaviour
+{
+    [SerializeField]
+    private Terrain terrain;
+    private Vector3 terrainSize;
+    private Vector3 camPosition;
+    private Transform cam;
+
+
+    // Start is called before the first frame update
```

Comment:
remove

Reply:
used for clarity, easier to understand cam.Translate than transform.Translate (potentially)